src/avgScrabbleScore.js

```diff
@@ -8,11 +8,15 @@ let scrabbleScore = require('./scrabbleScore');
  */
 function avgScrabbleScore(words) {
   // This is your job. :)
+  return scrabbleScore(words) / words.length;
```

Comment:
The idea is you'd pass in an array of words and get back their average score, while `scrabbleScore` takes a single word as input and gives you back its scrabble score.

If `words` is a string then `words.length` wouldn't be the number of words, it'd be the number of characters.

If `words` is an array then `scrabbleScore` is doing something surprising.